autotest-agent/src/nativeMain/kotlin/com/epam/drill/test/agent/configuration/Configuration.kt

```diff
@@ -39,24 +40,33 @@ actual object Configuration : AgentConfiguration {
         get() = configuration.value!!.parameters
 
     actual fun initializeNative(agentOptions: String) {
+        val environmentVariablesProvider = EnvironmentVariablesProvider()
+        logger.info { "initializeNative: Found environment variables: ${environmentVariablesProvider.configuration}" }

```

Comment:
Should lower logging level here from info to debug, cause it have been already done in other branch https://github.com/Drill4J/autotest-agent/pull/143